### DISCUSSION TITLE
关于近战单位的攻击距离

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#192] [Q&A]
在项目二文档中，明确指出近战单位和远程单位应该部署在不同的格子，这是主要区别。但没有对近战单位的攻击距离做出说明，他应该只能攻击位于同一格子的敌人，还是可以拥有一个明显小于远程单位的射程呢（类似明日方舟的近卫）

**Comment by liusy58** [comment]
都可以
emmm不用过于纠结这些细枝末节，没太大意义
